attributes/default.rb

```diff
@@ -18,4 +18,7 @@
 #
 
 default['build-essential']['compile_time'] = false
-default['build-essential']['msys']['path'] = "#{ENV['SYSTEMDRIVE']}\\msys"
+default['build-essential']['mingw']['prefix'] = "#{ENV['SYSTEMDRIVE']}\\mingw"
```

Comment:
Makes sense.  I picked `['build-essential']['mingw32']['path']` and `['build-essential']['mingw64']['path']` in case we ever needed more attributes to further customize the two toolchains.